packages/editor/src/components/post-trash/check.js

```diff
@@ -13,8 +13,12 @@ function PostTrashCheck( { isNew, postId, children } ) {
 
 export default withSelect( ( select ) => {
 	const { isEditedPostNew, getCurrentPostId } = select( 'core/editor' );
+	const { canUser } = select( 'core' );
+	const postId = getCurrentPostId();
+
 	return {
 		isNew: isEditedPostNew(),
-		postId: getCurrentPostId(),
+		canUserTrash: !! postId && canUser( 'delete', 'posts', postId ),
```

Comment:
Should the postType be hard-coded here? I'm seeing a 404 from the request that the resolver makes when trying to edit a reusable block. That might also be the cause of the test failures.

IIRC, the `rest_base` from the `postType` is the value that should be used.

Reply:
> Should the postType be hard-coded here? I'm seeing a 404 from the request that the resolver makes when trying to edit a reusable block. That might also be the cause of the test failures.
> 
> IIRC, the `rest_base` from the `postType` is the value that should be used.

Yep, I think you're correct on both accounts.